ops/terraform/services/server/server-load/security-groups.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_security_group" "this" {
```

Comment:
I personally like to see descriptive names for every resource, but I've seen that this is common pattern for the repo

Reply:
Resources with generic names like `this` is not original to the beneficiary-fhir-data repository. It's conventional, perhaps even idiomatic approach to terraform when there isn't an immediately obvious, more meaningful name for a given resource and/or the module defines just one resource of _this_ type. The `aws_security_group` resource here seems to satisfy both conditions as it's written now. 

`main` is also common and I've worked alongside some excellent engineers who preferred even shorter, more generic names like `x`. More conventional styles not only help us avoid bike shedding during code reviews, it's also a lighter cognitive load for those contributing, and resource definition snippets tend to be more re-usable when they remain generic. It can also be helpful under more exceptional scenarios like terraform yielding out-of-sync state through partial application during failure, when resources need to be imported for any number of reasons, and so on. I see that @mharvey has already gone forward with renaming it to `lambda` which is also _fine_, but let's not get carried away in dismissing conventional solutions in favor of more exotic names. After all, IaC shouldn't be _too_ exciting.